src/initial-submission/components/FileDetailsForm.test.tsx

```diff
@@ -154,7 +154,11 @@ describe('File Details Form', (): void => {
                             data: {
                                 uploadManuscript: {
                                     files: {
-                                        manuscriptFile: { url: 'http://localhost/file.pdf', filename: 'testfile.pdf' },
+                                        manuscriptFile: {
```

Comment:
would it be worth making "url" and "downloadLink" different values - just to ensure they don't get mixed up?

Reply:
reflected